py-polars/conftest.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+import polars
+
+
+@pytest.fixture(autouse=True)
```

Comment:
This is needed to import the `pl` in the code examples. Note that it lives in the root (`py-polars`), not one level deeper in `py-polars/tests`, otherwise it will not be picked up for the doctests.